scripts/ci/copyright_check.py

```diff
@@ -28,6 +28,18 @@ def get_header(has_shebang, regexes):
     indices = range(init_idx, init_idx + len(regexes))
     return zip(regexes, indices)
 
+# Matches all MODIFIED_HEADER patterns within the file. This is used
+# when the license is not at the top header, and there are licenses of
+# external libraries.
+def match_somewhere(regexes, lines):
+    maybe_match_head_index = [index for index, line in enumerate(lines) if regexes[0].search(line)][:1]
```

Comment:
Will this match adding the copyright header in the middle of the file?

Reply:
Yes it does. I’m not sure if there’s a good way to do check if our license is directly below the other license, but I can try to assert that all lines above start with a comment or is empty. Let me know if that works.

Reply:
Implemented proposal with https://github.com/model-checking/kani/pull/1447/commits/f4665b06a3fde09a1a496f35fc8e663dff296cfa

Feel free to reopen if it is not what you intended.